src/encoded/audit/analysis.py

```diff
@@ -115,13 +376,23 @@ def audit_experiment_standards_dispatcher(value, system, files_structure):
         'ENCODE2-Mouse',
         'ENCODE2',
         'ENCODE',
-        'Roadmap'
+        'Roadmap',
+        'modERN'
     ]:
         return
+    # Allow White lab for Transcription factor ChIP-seq pipeline (modERN) and
+    # Ruan lab for ChIA-PIPE
     if len(value['pipeline_labs']) != 1 or \
-            value['pipeline_labs'][0] != '/labs/encode-processing-pipeline/':
+            value['pipeline_labs'][0] not in [
+                '/labs/encode-processing-pipeline/',
+                '/labs/kevin-white/',
+                '/labs/yijun-ruan/'
+    ]:
         return
```

Comment:
I'm wondering if this will exclude something we do want? I've seen this in the old version but it doesn't seem to be for all assays. Also, maybe the pipeline title check below will ensure proper auditing so we don't need this check here?